docs/metadata.rst

```diff
@@ -106,6 +106,15 @@ geospatial and temporal ranges of entries within a data set. This is a complex s
 handled by DCAT. If you need to express this kind of information, please do so according to the standards laid out
 `here <https://www.w3.org/TR/vocab-dcat-2/#time-and-space>`_.
 
+We encourage use of the ``dcat:keyword`` list for data sets. These translate to "tags" in our web interface and are useful to group data sets around specific topics.
+
+.. code-block:: yaml
+
+  dcat:keyword: 
+    solar
+    electricity
+    retrofit
```

Comment:
I'm fairly sure that in YAML each list item needs a `- ` prefix so this won't parse correctly.  However, YAML is a complicated format and I could easily be wrong.

Also, when I look at our [metadata parser's source code](https://github.com/icebreakerone/open-energy-python-infrastructure/blob/72eddd2d09a2f09b7383d4a9939c8fd035b56b2f/src/python/ib1/openenergy/support/metadata.py#L120) I wonder if this should be `dcat:keywords` (plural, not singular).

If you've tested this example and seen it work, then please ignore my concerns.

Reply:
Thanks @tomhukins. Good spot on the YAML - It passed the YAML parser in Sphinx which lulled me into a false sense of security, but it is incorrect for a YAML list. I've pushed a fix to this branch - please double-check.

The [correct DCAT property is keyword](https://www.w3.org/TR/vocab-dcat-3/#Property:resource_keyword) (singular). Confusingly the metadata parser parses `keyword` to its own property `keywords`. I can confirm the singular version has been fetched correctly to our index and converted to CKAN tags.